src/icon-manifest/supportedFolders.ts

```diff
@@ -66,6 +66,7 @@ export const extensions: IFolderCollection = {
     { icon: 'mjml', extensions: ['mjml', '.mjml'], format: FileFormat.svg },
     { icon: 'model', extensions: ['models', '.models'], format: FileFormat.svg },
     { icon: 'mongodb', extensions: ['mongodb'], format: FileFormat.svg },
+    { icon: 'nginx', extensions: ['nginx', 'conf.d'], format: FileFormat.svg },
```

Comment:
is `conf.d` dedicated for nginx?

Reply:
https://github.com/vscode-icons/vscode-icons/issues/1404#issuecomment-352196284